src/Tecan/Rack/Rack.php

```diff
@@ -4,7 +4,20 @@
 
 interface Rack
 {
+    /**
+     * Source labware barcode.
+     */
+    public function id(): ?string;
+
+    /**
+     * User-defined label (name) which is assigned to the source labware.
+     */
     public function name(): string;
 
+    /**
+     * Source labware type (configuration name), e.g. “384 Well, landscape”.
+     */
     public function type(): string;
+
+    public function toString(): string;
```

Comment:
Probably a good idea to describe the purpose of the string representation - is it debugging?